packages/shorebird_code_push_protocol/test/src/models/registry_test.dart

```diff
@@ -44,14 +44,14 @@ void main() {
                   Release(version: '1.0.1'),
                 ],
               ),
-              App(productId: 'app2'),
+              App(id: 'app2'),
             ],
           ),
           Account(
             apiKey: 'api_key2',
             apps: [
               App(
-                productId: 'app2',
+                id: 'app2',
```

Comment:
So this may not actually be the "id" in the database.  Since this is a per-account id, not a global-unique id.